cmd/dataset/app/dataset.go

```diff
@@ -121,14 +123,16 @@ func handle() {
 
 	utils.NewPprofServer(setupLog, pprofAddr, development)
 
+	// the default webserver port is 9443, no need to set.
 	mgr, err := ctrl.NewManager(controllers.GetConfigOrDieWithQPSAndBurst(kubeClientQPS, kubeClientBurst), ctrl.Options{
-		Scheme:                  scheme,
-		MetricsBindAddress:      metricsAddr,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: metricsAddr,
+		},
 		LeaderElection:          enableLeaderElection,
 		LeaderElectionNamespace: leaderElectionNamespace,
 		LeaderElectionID:        "dataset.data.fluid.io",
-		Port:                    9443,
-		NewCache:                NewCache(scheme),
+		Cache:                   NewCacheOptions(),
```

Comment:
Why using `Cache` instead of `NewCache`?

Reply:
Previous used `cache.BuilderWithOptions` function is removed. The `NewCacheFunc` is changed to LOW LEVEL PRIMITIVE! We only need to configure the cache options, no need to re-implement a cache function.

Reply:
Got it. Thanks.